msteams-platform/apps-in-teams-meetings/build-tabs-for-meeting.md

```diff
@@ -176,6 +176,129 @@ TeamsJS is a rich library used to create Tabs using JavaScript. Use the latest T
+
+The following is the flow diagram of the launch of cached app:
+
+:::image type="content" source="../assets/images/saas-offer/cached-launch-app.png" alt-text="This screenshot shows the flow of the cached launch of the app in meeting side panel.":::
+
+When you opt into app caching, the webview that is used to host the embedded app is reused as users navigate to different instances of the app within a window. The webview used to host the app is hidden when the users leave the app and shown when the users return to the app. When the app is cached, any audio that is playing is muted.
+
+> [!NOTE]
+> If the app caching isn't enabled, the webview is recreated every time the users leave and return to the app.
+
+Following are the parameters to control the conditions for the apps to be added or removed from the cache (based on the configuration updates, the parameters can be modified):
+
+* If the system memory load is high, the app is removed from the cache.
+* If the app exceeds maximum cache size, the app is removed from the cache.
+* If the user doesn't return to the app within 20 minutes, the app is removed from the cache.
+* The app is not cached if Teams doesn't receive the `readyToUnload` signal from the app within 30 seconds after sending the `beforeUnload` notification.
+* App caching is disabled if the system memory is less than 4 GB.
+* Side panel is the only supported FrameContext for app caching in meetings.
+* App caching isn't supported for meetings where the invited user count is more than 20.
+
+### Code example
+
+The following code snippet is an example for `readyToUnload` callback function:
+
+```javascript
+// Using readyToUnload callback function
+const beforeUnloadHandler = (readyToUnload: () => void) => {
+  readyToUnload();
+  return true;
+};
+```
+
+The following code snippet is an example of `teamsCore.registerBeforeUnloadHandler` and `teamsCore.registerOnLoadHandler` APIs:
+
+```javascript
+microsoftTeams.app.initialize().then(() => {
+
+    microsoftTeams.teamsCore.registerBeforeUnloadHandler((readyToUnload) => {
+        const result = beforeUnloadHandler(readyToUnload);
+        // dispose resources and then invoke readyToUnload
+        console.log("got beforeunload from TEAMS");
+        return result;
+    });
+
+    microsoftTeams.teamsCore.registerOnLoadHandler((data) => {
+        console.log("got load from TEAMS", data.contentUrl, data.entityId);
+        // use contentUrl to route to correct page 
+        // invoke notifySuccess when ready  
+        app.notifySuccess();
+    });
+});
+```
+
+### Limitations
+
+The following are the limitations for app caching:
+
+* Single-page apps that use client-side routing for page navigation can benefit from app caching. It's recommended that the same domain is used across all contexts of your app launch.
```

Comment:
same domain be used

Reply:
updated as suggested

Reply:
Just a note here: it's not singular vs plural. Rather, "be used" is in the subjunctive mood (https://www.merriam-webster.com/words-at-play/getting-in-the-subjunctive-mood), versus "is used", which is in the indicative. Since the verb in the main clause is "recommended", it's appropriate to use the subjunctive.